src/components/expanded-state/chart/chart-data-labels/ChartPriceLabel.js

```diff
@@ -89,7 +90,7 @@ export default function ChartPriceLabel({
           return formatted;
         }}
         style={{
-          width: '100%',
+          width: android ? '67%' : '75%',
         }}
```

Comment:
same, you can pass a prop and use it to write the css dynamically in the styled component